report/src/main/webapp/app/services/endpointConfiguration.service.js

```diff
@@ -18,11 +18,18 @@
 define(['angularAMD'], function (angularAMD) {
   'use strict';
   angularAMD.factory('endpointConfiguration', function () {
+    var domain = 'http://aet-vagrant';
     return {
       getEndpoint: function () {
         var config = {
           //'getUrl': '/api/'
-          'getUrl': 'http://aet-vagrant/api/'
+          'getUrl': domain + '/api/'
+        };
+        return config;
+      },
+      getDomain: function () {
+        var config = {
+          'getUrl': domain
         };
```

Comment:
This change is a result of change `suitestatus` to `api/suitestatus`. After that servlet returns `api/..`, and after concatenation I will have `/api/api/suitestatus` -> incorrect URL. I can escape `api` from one of this URL, but I guess it isn't good practice. In additional I didn't want to manipulate URL returned by the servlet, it could be problematic for the client and create more problems for this easy case so I decided to add the new function with a used domain from a variable. Is it a good way? Will it require from users some changes in their configs?

Reply:
Please look at my review comment. This can't be programmed like now.